instrumentation/jmx-metrics/library/jetty.md

```diff
@@ -0,0 +1,40 @@
+# Jetty Metrics
+
+Here is the list of metrics based on MBeans exposed by [Jetty](https://jetty.org/).
+
+The metrics captured and their respective attributes depend on the Jetty version:
+- [Jetty 12 and later](#jetty-12-and-later)
+- [Jetty 9 to 11](#jetty-9-to-11)
+
+## Jetty 12 and later
+
+Those metrics require the following Jetty modules to be enabled : `jmx`, `http`, `statistics`, `sessions` and at least one of `ee8-deploy`, `ee9-deploy` or `ee10-deploy`.
+
+| Metric Name             | Type          | Attributes    | Description                               |
+|-------------------------|---------------|---------------|-------------------------------------------|
+| jetty.thread.count      | UpDownCounter |               | The current number of threads             |
```

Comment:
For stable metrics, such as the jvm one, description ends with a dot.

Reply:
yeah, we should make up our mind in semconv 😄 

I've opened https://github.com/open-telemetry/semantic-conventions/issues/2505

let's just merge as-is, there will be cleanup needed one way or the other once that decision is made